src/core/a-animation.js

```diff
@@ -373,6 +379,19 @@ function getAnimationValues (el, attribute, dataFrom, dataTo, currentValue) {
   };
 
   /**
+   * Match the scema type to color
+   * @return {bool} if the schema is of type color
+   */
+  function isColor () {
+    var componentName = attributeSplit[0];
+    var propertyName = attributeSplit[1];
+    var component = el.components[componentName];
+    var schema = component && component.schema;
+    if (schema) { return schema[propertyName].type === 'color'; }
```

Comment:
This throws an error when schema[propertyName] is undefined so it would have to be:

```
  return schema && schema[propertyName] && schema[propertyName].type === 'color';
```

Would protect from all. Not sure how often the schema is undefined I think its only when given bad input, but probably better than letting an error get thrown?